dll/win32/imm32/utils.c

```diff
@@ -793,11 +799,17 @@ BOOL APIENTRY Imm32LoadImeVerInfo(PIMEINFOEX pImeInfoEx)
 
     cbVerInfo = s_fnGetFileVersionInfoSizeW(szPath, &dwHandle);
     if (!cbVerInfo)
+    {
+        ERR("!cbVerInfo\n");
         goto Quit;
+    }
 
     pVerInfo = ImmLocalAlloc(0, cbVerInfo);
     if (!pVerInfo)
+    {
+        ERR("!pVerInfo\n");
```

Comment:
Not sure of the usefulness of those dprints for the long run... (and for debugging in a more general way, since correct debugging should be done with a debugger, not by looking at prints).

Reply:
Deleted.